### PR TITLE
Testing new checkboxes for the plugin manager

### PR DESCRIPTION
**Pull request by NotMyFault**
https://github.com/jenkinsci/jenkins/pull/6527

### CODE REVIEW

**Comment by NotMyFault** [comment]
cc @janfaracik

**Comment by timja** [comment]
Defo needs some changes 😄

**Comment by janfaracik** [comment]
Should be all sorted now, essentially the new checkbox requires the `input` be wrapped in a `span` and that caused the selector to break. Fixed it by using `ancestor` now to get the parent row so it'll work no matter how many times the `input` is wrapped.

File: src/main/resources/ath-container/run.sh [hunk 1]
```diff
@@ -44,8 +44,8 @@ if [ ! -f $war ]; then
     mirrors=https://get.jenkins.io
     case "$war" in
         "latest")
-            war=jenkins-latest.war
-            url=$mirrors/war/latest/jenkins.war
+            #war=jenkins-latest.war
+            url=https://ci.jenkins.io/job/Core/job/jenkins/job/PR-6527/lastSuccessfulBuild/artifact/org/jenkins-ci/main/jenkins-war/2.347-rc32357.891f43b_fd2c4/jenkins-war-2.347-rc32357.891f43b_fd2c4.war
```
Comment (timja) [inline src/main/resources/ath-container/run.sh:48]:
```suggestion
            war=jenkins-latest.war
            url=$mirrors/war/latest/jenkins.war
```

**Review by timja — APPROVED** [review]

**Pull request merged**